### DISCUSSION TITLE
Latest April 1, 2023 Dev. build not working in Windows

### DISCUSSION THREAD

**Discussion by tdh1967** [official-stockfish/Stockfish#4486] [General]
After creating the engine in Fritz GUI, I go to change the main engine and the latest April 1 version of StockFish just created isn't on the list.

**Comment by vondele** [comment]
Can you start the engine from the command line ? If not, what's the error message, if yes, can you type at the prompt `go depth 5` ?

**Reply by tdh1967** [reply]
Thanks, but this is the first time a DEV version I've installed doesn't appear as an engine after being created. No error messages either - it just doesn't show as an installed engine. I had a similar issue with a Brainlearn version two years ago and the author fixed it as it was a problem with the program and wouldn't run in the GUI for anyone. I appreciate your response, but if no one else is facing this in Chessbase 17 GUI, I'll wait for the next update. It might run from the prompt, but I prefer my usual GUI. All previous versions over the years, even lately, worked fine. If I'm the only one with this problem, it's a first for me with StockFish..

**Reply by pfalcon1949** [reply]
I faced this exact problem today. i tried many different versions on the site all dated April 1 and all do not work. Tried a Mar 29 build and it works fine. I feel this is some kind of April fools joke and I am not sure. I also tried some older versions like the stockfish15-1 which works fine. I tried all this within Chessbase 17 interface by the way. Hope that helps

**Reply by shadizahran** [reply]
Dear @vondele 
The reason that Stockfish engine couldn't be installed on Fritz 18or Chessbase 17 is due to the quotation marks included newly in the engine name """.
I advise to set the engine name to Stockfish ver-xxxxxxx-xxxxxx  like before April 1st (without "xxxxx"-"xxxxx")

**Reply by pfalcon1949** [reply]
removing quotation marks gives error right away. I still say it is April fool's joke!

**Reply by pfalcon1949** [reply]
Try the second or third in the list of development builds. They are now working fine. Only the very first one which has the notorious quotation marks (removing them is no solution!) is not working!

**Comment by OmegaThree** [comment]
I have the same problem with the Fritz GUI, It runs fine from the engine command line in windows but will not show in Fritz. This may have something to do with the exclamation marks within the name 

Stockfish dev-"20230401"-"6a6e32df" by the Stockfish developers (see AUTHORS file)

Previous stockfish that works fine

Stockfish dev-20230329-3f01e3f4 by the Stockfish developers (see AUTHORS file)

**Reply by dav1312** [reply]
https://github.com/official-stockfish/Stockfish/commit/66bf45b99e2061c1ba74f9975bc5059ac0121dfd

**Reply by vondele** [reply]
That's strange... there are no `"` (quotes) in the output here.

**Reply by vondele** [reply]
but the abrok binary shows them ...

**Reply by vondele** [reply]
the github action binaries are also fine.

**Reply by vondele** [reply]
I've written the abrok owner to have a look

**Reply by vondele** [reply]
I've got a reply, will be fixed probably tonight.

**Reply by tdh1967** [reply]
I just saw this.  Thanks for posting.  Hope to see a fix.

**Comment by vondele** [comment]
will probably be fixed tonight.

**Reply by tdh1967** [reply]
Appreciate the info.

**Comment by francoal** [comment]
The new Stockfish does not work when activated with ChessBase or Fritz 18. I activated it with Arena and it works

**Reply by vondele** [reply]
an issue with the abrok compiled binaries.
You can download latest binaries on github as well https://github.com/official-stockfish/Stockfish/actions/runs/4583679966 (look for artifacts), they are fine.

**Reply by shadizahran** [reply]
Dear @vondele  
The reason that Stockfish engine couldn't be installed on Fritz 18or Chessbase 17 is due to the quotation marks included newly in the engine name """.
I advise to set the engine name to Stockfish ver-xxxxxxx-xxxxxx  like before April 1st (without "xxxxx"-"xxxxx")

**Comment by vondele** [comment]
The abrok issue has been fixed. Downloads now are OK.
